website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

```diff
@@ -33,6 +33,31 @@ to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowfl
 **Note**: The schema field in the **Developer Credentials** section is a required field.
 <Lightbox src="/img/docs/dbt-cloud/snowflake-userpass-auth.png" width="70%" title="Snowflake username/password authentication"/>
 
+### Snowflake MFA
+
+**Prerequisites:**
+- A Deployment or Development environment in a dbt Cloud project
+- The Duo authentication app
+- Admin access to Snowflake to configure MFA settings for the account
```

Comment:
```suggestion
- Admin access to Snowflake if MFA  settings have not already been applied to the account
```

Reply:
Can you remove the deployment environment option? I don't want to say it's not supported (because it is), but MFA should really only be used in development

Apart from that, we're golden!

Reply:
will get the removed and then merge!